src/community/script/py/src/main/java/org/geoserver/script/py/PyAppHook.java

```diff
@@ -239,30 +201,30 @@ public void write(OutputStream outputStream) throws IOException {
      *
      * @throws IOException 
      */
-    PyObject createEnviron(Request request) throws IOException {
+    PyObject createEnviron(HttpServletRequest request) throws IOException {
         
         PyDictionary environ = new PyDictionary();
         
         environ.put("REQUEST_METHOD", request.getMethod().toString());
         
-        Reference ref = request.getResourceRef();
-        environ.put("SCRIPT_NAME", ref.getLastSegment());
+//        Reference ref = request.getResourceRef();
+//        environ.put("SCRIPT_NAME", ref.getLastSegment());
 
         //force to pystring so that frameworks don't try to encode as idna 
-        environ.put("SERVER_NAME", new PyString(ref.getHostDomain()));
-        environ.put("SERVER_PORT", String.valueOf(ref.getHostPort()));
-
-        List<String> seg = new ArrayList(ref.getSegments().subList(4, ref.getSegments().size()));
-        seg.add(0, "");
-        
-        environ.put("PATH_INFO", StringUtils.join(seg.toArray(), "/"));
-        //environ.put("PATH_INFO", );
-        
-        environ.put("QUERY_STRING", request.getResourceRef().getQuery());
-
-        environ.put("wsgi.version", new PyTuple(new PyInteger(0), new PyInteger(1)));
-        environ.put("wsgi.url_scheme", ref.getScheme());
-        environ.put("wsgi.input", new PyFile(request.getEntity().getStream()));
+//        environ.put("SERVER_NAME", new PyString(ref.getHostDomain()));
+//        environ.put("SERVER_PORT", String.valueOf(ref.getHostPort()));
+//
+//        List<String> seg = new ArrayList(ref.getSegments().subList(4, ref.getSegments().size()));
+//        seg.add(0, "");
+//
+//        environ.put("PATH_INFO", StringUtils.join(seg.toArray(), "/"));
+//        //environ.put("PATH_INFO", );
+//
+//        environ.put("QUERY_STRING", request.getResourceRef().getQuery());
+//
+//        environ.put("wsgi.version", new PyTuple(new PyInteger(0), new PyInteger(1)));
+//        environ.put("wsgi.url_scheme", ref.getScheme());
+//        environ.put("wsgi.input", new PyFile(request.getEntity().getStream()));
```

Comment:
Why is this commented out? If it is unnecessary, remove it.

Reply:
fixed